.make-packages.js

```diff
@@ -112,10 +112,10 @@ fs.copySync('src/testing/package.json', PKG_ROOT + '/testing/package.json');
 if (fs.existsSync(UMD_ROOT)) {
   fs.copySync(UMD_ROOT, UMD_PKG);
   // Add licenses to tops of bundles
-  addLicenseToFile('LICENSE.txt', UMD_PKG + 'Rx.js');
-  addLicenseTextToFile(license, UMD_PKG + 'Rx.min.js');
-  addLicenseToFile('LICENSE.txt', UMD_PKG + 'Rx.js');
-  addLicenseTextToFile(license, UMD_PKG + 'Rx.min.js');
+  addLicenseToFile('LICENSE.txt', UMD_PKG + 'rx.js');
```

Comment:
I suggest that all these files are called `rxjs.*`, i.e. `rxjs.umd.js` and `rxjs.umd.min.js`

Reply:
this is just to be consistent about the naming across all the namespaces.